src/components/general/Navbar/Notifica/Notifications/index.js

```diff
@@ -144,6 +143,8 @@ const NotificationEmpty = () => (
 );
 
 const notificationKeySelector = n => n.id;
+
+// FIXME: this is probably breaking change
```

Comment:
@AdityaKhatri needs to find a way to do this without listening to global requests